packages/calcite-components/src/utils/time.ts

```diff
@@ -167,19 +216,50 @@ export function getMeridiem(hour: string): Meridiem {
   return hourAsNumber >= 0 && hourAsNumber <= 11 ? "AM" : "PM";
 }
 
-export function getMeridiemOrder(locale: string): number {
-  const isRtl = locale === "ar" || locale === "he";
-  if (isRtl) {
-    return 0;
+export function getMeridiemFormatToken(locale: SupportedLocale): "a" | "A" | "a " | " a" | " A" | "A " {
+  const localizedAM = getLocalizedMeridiem(locale, "AM");
+  const localizedPM = getLocalizedMeridiem(locale, "PM");
+  const meridiemOrder = getMeridiemOrder(locale);
+  let separator: "" | " ";
+  const timeParts = getTimeParts({
+    hour12: true,
+    value: "00:00:00",
+    locale,
+    numberingSystem: "latn",
+  });
+  if (meridiemOrder === 0) {
+    separator = timeParts[1].type === "hour" ? "" : " ";
+  } else {
+    separator = timeParts[meridiemOrder - 1].type === "second" ? "" : " ";
+  }
+  // Unknown dayjs parsing bug with norwegian.  Dayjs only accepts uppercase meridiems for some reason, despite the LT/LTS config
```

Comment:
No, because we're [using `nb` under the hood anyway.](https://github.com/Esri/calcite-design-system/blob/5f97c55b50d3295f298fcccea5603115a8cdb279/packages/calcite-components/src/components/input-time-picker/input-time-picker.tsx#L698)  I wasn't able to track down in the dayjs source where this issue is happening, so I just worked around it.